### PR TITLE
Feature/hello sdl

### PR DESCRIPTION
**Pull request by brettywhite**
Added Documentation for Hello SDL Android.

### CODE REVIEW

**Comment by askirk** [comment]
Should we add a section for usb transport? Select `usbDebug` build variant, run, plug into TDK using USB, accept the accessory connect dialog on the phone.

**Comment by brettywhite** [comment]
@askirk @joeygrover is that necessary for the hello sdl android app? What use cases besides nav or video streaming (both not actually supported yet, as far as i know) would this come up for? I'm not opposed to it, I just don't want the first guide a developer uses for SDL to be mucked up with stuff they aren't concerned with yet.

**Comment by joeygrover** [comment]
I believe a section at the end of "Other Transports" could be useful. At least how to setup the HelloSdl app to use those different transports, not setting up core to use them.

File: docs/Getting Started/Hello Sdl Android/index.md [hunk 1]
```diff
@@ -57,9 +59,13 @@ In the top of this file, locate the variable declaration for ```DEV_MACHINE_IP_A
...
 
-### Configure for Bluetooth
+#### Configure for Bluetooth
+
+Right out of the box, all you need to do to run bluetooth is to select the ```mbt_offDebug``` (Multiplexing Bluetooth) build flavor.
+
+#### Configure for USB (AOA)
 
-Right out of the box, all you need to do to run bluetooth is to select the ```mbt_offDebug``` (Multiplexing Bluetooth) build flavor. 
+If you modify Hello Sdl Android to need USB transport, select the ```usbDebug``` build flavor. There is also much more information for USB transport under Getting Started - Using AOA Protocol.
```
Comment (askirk) [inline docs/Getting Started/Hello Sdl Android/index.md]: "If you modify Hello Sdl Android to need USB transport" to "To connect to a TDK via USB transport" 

and

"There is also much more information" to "There is more information"

File: docs/Getting Started/Hello Sdl Android/index.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (joeygrover) [inline docs/Getting Started/Hello Sdl Android/index.md]: Why all this whitespace?

File: docs/Getting Started/Hello Sdl Android/index.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+!!!
+
+At this point Hello Sdl Android has been compiled and is running properly! Continue reading through our guides to learn about all of the 
+RPCs (Remote Procedure Calls) that can be made with the library. 
+
+### Troubleshooting
+
+Sometimes things don't always go as planned, and so this section exists. If your app compiles and does NOT show up on the HMI, there are a few things to check out.
+
+#### TCP
```
Comment (joeygrover) [inline docs/Getting Started/Hello Sdl Android/index.md]: We don't need TCP instructions again.

File: docs/Getting Started/Hello Sdl Android/index.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+Lock Screens are an important part of Sdl enabled applications. The goal is to keep the driver's eyes forward and off of the device
+!!!
+
+At this point Hello Sdl Android has been compiled and is running properly! Continue reading through our guides to learn about all of the 
+RPCs (Remote Procedure Calls) that can be made with the library. 
+
+### Troubleshooting
+
+Sometimes things don't always go as planned, and so this section exists. If your app compiles and does NOT show up on the HMI, there are a few things to check out.
+
```
Comment (joeygrover) [inline docs/Getting Started/Hello Sdl Android/index.md]: Should add a header like `Other Transports`

File: docs/Getting Started/Hello Sdl Android/index.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+#### TCP
+
+1. Make sure that you have changed the IP in ```SdlService.java``` to match the machine running Sdl Core. Being on the same network is also important.
+2. If you are sure that the IP is correct and it is still not showing up, make sure the Build Flavor that is running is ```tcpDebug```.
+3. If the two above dont work, make sure there is no firewall blocking the incoming port ```12345``` on the machine or VM running SDL Core. In the same breath, make sure your firewall allows that outgoing port. 
+
```
Comment (joeygrover) [inline docs/Getting Started/Hello Sdl Android/index.md]: Add USB/AOA transport

File: docs/Getting Started/Hello Sdl Android/index.md (hunk 1 above)
Comment (joeygrover) [inline docs/Getting Started/Hello Sdl Android/index.md]: `To connect to an SDL Core instance or TDK via USB transport` and add a note that the sdl core instance might also need separate configurations.

**Pull request merged**